app/controllers/messages_controller.rb

```diff
@@ -82,4 +84,12 @@ def require_owner
 	  end
   end
 
+  def message_params
+    params.require(:message).permit(:to_id, :subject, :body).merge(
+      from_id: current_user.id,
+	    user_id: current_user.id,
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Tab detected.